.github/workflows/release-t0rn.yml

```diff
@@ -0,0 +1,251 @@
+name: t0rn Rococo Release
+
+# Trigger only on push to this branch
+on:
+  push
+
+# TODO: uncomment after tests
+# on:
+#   push:
+#     tags:
+#       - "v[0-9]+.[0-9]+.[0-9]+-rc.[0-9]+"
+
+env:
+  PARACHAIN_NAME: t0rn
+  RUST_BACKTRACE: full
+  CARGO_TERM_COLOR: always
+
+jobs:
+  build-release:
+    runs-on: ["self-hosted", "ep1c"]
+    steps:
+      - name: ☁ Checkout git repo
+        uses: actions/checkout@v3
+        with:
+          submodules: recursive
+          token: ${{ secrets.GH_PAT }}
+
+      - name: ⚙️ Get nightly rust toolchain with wasm target
+        uses: actions-rs/toolchain@v1
+        with:
+          toolchain: nightly-2022-06-16
+          target: wasm32-unknown-unknown
+          components: rustfmt, clippy
+          override: true
+
+      - name: 🕒 Cache Rust binaries and packages
+        uses: actions/cache@v2
+        id: cache-rust
+        with:
+          path: |
+            ~/.cargo/registry
+            ~/.cargo/git
+            ./target
+          key: ${{ runner.os }}-cargo-${{ hashFiles('./Cargo.lock') }}
+
+      - name: 🏭 Build circuit
+        uses: actions-rs/cargo@v1
+        continue-on-error: false
+        with:
+          command: build
+          args: --manifest-path ./node/${{ env.PARACHAIN_NAME }}-parachain/Cargo.toml --locked --release
+
+      - name: 📤 Upload collator binary
+        uses: actions/upload-artifact@v2
+        with:
+          name: ${{ github.sha }}-${{ env.PARACHAIN_NAME }}-collator-release-artifacts
+          path: ./target/release/${{ env.PARACHAIN_NAME }}-collator
+          if-no-files-found: error
+          retention-days: 1
+
+      - name: 🫧 Build runtime WASM
+        run: |
+          ./scripts/build_wasm.sh ${{env.PARACHAIN_NAME}}
```

Comment:
this shouldnt be run on release only. It should be run on changelog generation(on all merges into trunk)